app/controllers/organizations/adopter_fosterer/adopted_pets/tasks_controller.rb

```diff
@@ -0,0 +1,24 @@
+module Organizations
+  module AdopterFosterer
+    module AdoptedPets
+      class TasksController < Organizations::BaseController
+        before_action :context_authorize!
+
+        def index
+          @pet = Pet.find(params[:adopted_pet_id])
+          @tasks = @pet.tasks.where(completed: false)
```

Comment:
We can use this scope I think
https://github.com/rubyforgood/pet-rescue/blob/007914e4be525f35e245cee9bb909f3af052ddb2/app/models/task.rb#L35

Reply:
Yeah, I missed this scope.. Have added it now.